tests/distro-regression/tests/tests.yaml

```diff
@@ -7,9 +7,9 @@ smoke_bundles:
   - keystone_v3_smoke: xenial-queens
   - keystone_v3_smoke: bionic-queens
   - keystone_v3_smoke_rocky: bionic-rocky
-  - keystone_v3_smoke: bionic-stein
-  - keystone_v3_smoke: bionic-train
-  - keystone_v3_smoke: bionic-ussuri
+  - keystone_v3_smoke_octavia: bionic-stein
+  - keystone_v3_smoke_octavia: bionic-train
+  - keystone_v3_smoke_octavia: bionic-ussuri
```

Comment:
Sorry for the nitpick. Could we call this keystone_v3_smoke_stein? It aligns better with the other model alias names.